rpc_jobs/multi_node_aio.yml

```diff
@@ -49,9 +157,9 @@
         stage('OnMetal Multi-Node AIO') {{
           git branch: "{branch}", url: "https://github.com/rcbops/rpc-openstack"
           if("{series}" == "mitaka"){{
-            build job: 'OnMetal-Multi-Node-AIO_mitaka-trusty-post-merge'
+            build job: 'OnMetal-Multi-Node-AIO_mitaka-trusty-deploy-post-merge'
```

Comment:
Is only mitaka enough here?
Not so sure to understand the prupose of all of this.